zubhub_frontend/zubhub/src/views/activity_details/ActivityDetailsV2.jsx

```diff
@@ -135,7 +148,15 @@ export default function ActivityDetailsV2(props) {
               </Typography>
             </div>
           </div>
-          <AnchorElemt isLoading={isLoading.delete} onDelete={handleDelete} onEdit={handleEdit} />
+          {props.auth.tags.includes('staff') && (
```

Comment:
Only staff users can see the `edit`, `delete`, and `publish/unpublish`

Reply:
Activity creators should be able to see edit and delete as well

Reply:
You mean the one with `creator` tag?

Reply:
Resolved. Thanks